app/assets/stylesheets/main.scss

```diff
@@ -1,290 +1,290 @@
-text-align:center;
-background-color:#d0d0d0;
-margin:0;
-padding:1em 0;
+  text-align:center;
+  background-color:#d0d0d0;
+  margin:0;
+  padding:1em 0;
 }
 
 #frontpage_examples div#examples_0 {
-float:left;
-margin-left:0;
-width:49%;
+  float:left;
+  margin-left:0;
+  width:49%;
 }
 
 #frontpage_examples div#examples_1 {
-float:right;
-margin-right:0;
-width:49%;
+  float:right;
+  margin-right:0;
+  width:49%;
 }
 
 #frontpage_examples ul {
-text-align:center;
-list-style:none;
-margin:0;
-padding:1em 0;
+  text-align:center;
+  list-style:none;
+  margin:0;
+  padding:1em 0;
 }
 
 div#twitter {
-margin-top: 30px;
+  margin-top: 30px;
 }
 
 img.twitter-icon {
-vertical-align:middle;
+  vertical-align:middle;
 }
 
 .blog_post {
-margin-bottom:2em;
+  margin-bottom:2em;
 }
 
 .request_listing,.body_listing,.user_listing {
-font-size:0.9em;
-margin-top:1.5em;
-border-bottom:#9C9C9C;
-overflow:hidden;
-border-style:none none solid;
-border-width:0 0 1px;
-padding:0 0 1.5em;
+  font-size:0.9em;
+  margin-top:1.5em;
+  border-bottom:#9C9C9C;
+  overflow:hidden;
+  border-style:none none solid;
+  border-width:0 0 1px;
+  padding:0 0 1.5em;
 }
 
 .body_listing {
-_width:47em;
-padding-bottom:16px;
+  _width:47em;
+  padding-bottom:16px;
 }
 
 span.head {
-display:block;
-font-size:1.4em;
-font-weight:700;
-padding:12px 0 0;
+  display:block;
+  font-size:1.4em;
+  font-weight:700;
+  padding:12px 0 0;
 }
 
 .request_listing span.head /* full page request list only */ {
-min-height:32px;
-background-image:image-url('navimg/request-icon.png');
-background-repeat:no-repeat;
-background-position:4px 0;
-margin:0 0 0.3em;
-padding:8px 0 0 42px;
+  min-height:32px;
+  background-image:image-url('navimg/request-icon.png');
+  background-repeat:no-repeat;
+  background-position:4px 0;
+  margin:0 0 0.3em;
+  padding:8px 0 0 42px;
 }
 
 .body_listing span.head /* full page request list only */ {
-min-height:32px;
-background-image:image-url('navimg/auth-icon.png');
-background-repeat:no-repeat;
-background-position:4px 0;
-margin:0 0 0.3em;
-padding:8px 0 0 42px;
+  min-height:32px;
+  background-image:image-url('navimg/auth-icon.png');
+  background-repeat:no-repeat;
+  background-position:4px 0;
+  margin:0 0 0.3em;
+  padding:8px 0 0 42px;
 }
 
 .user_listing span.head /* full page request list only */ {
-min-height:32px;
-background-image:image-url('navimg/user-icon.png');
-background-repeat:no-repeat;
-background-position:4px 0;
-margin:0 0 0.3em;
-padding:8px 0 0 42px;
+  min-height:32px;
+  background-image:image-url('navimg/user-icon.png');
+  background-repeat:no-repeat;
+  background-position:4px 0;
+  margin:0 0 0.3em;
+  padding:8px 0 0 42px;
 }
 
 .user_listing span.no_icon /* full page request list only */ {
-background-image:none;
+  background-image:none;
 }
 
 span.bottomline {
-clear:left;
-display:block;
-padding:10px 0 0 42px;
+  clear:left;
+  display:block;
+  padding:10px 0 0 42px;
 }
 
 .request_listing span.bottomline
```

Comment:
Ah, a few oddly positioned comments broke the formatter.  

Fixed in 1ee0cdef2a92e068494dae93c723061f5abf5e86  


On Tuesday, 21 July 2015 at 14:50, Louise Crow wrote:

> In app/assets/stylesheets/main.scss (https://github.com/mysociety/alaveteli/pull/2671#discussion_r35103263):
> > } > > span.bottomline { > -clear:left; > -display:block; > -padding:10px 0 0 42px; > + clear:left; > + display:block; > + padding:10px 0 0 42px; > } > > .request_listing span.bottomline  
> From here on, the css block contents aren't indented.
>  
> —
> Reply to this email directly or view it on GitHub (https://github.com/mysociety/alaveteli/pull/2671/files#r35103263).
>  
>  
>  
>  
>